draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -239,6 +273,46 @@
             staying within the maximum character transmission rate (<xref target="sec.sdp.dcsa"/>),
             if such has been provided by the peer.
           </t>
+          <t>
+            An implementation needs to take the user requirements for smooth flow and low
+            latency in real-time text conversation into consideration when assigning a buffer time.
+            It is RECOMMENDED to use the default transmission interval of 300 milliseconds
+            <xref target="RFC4103"/>, or lower, also for T.140 data channels.
+          </t>
+        </section>
+        <section anchor="sec.t140.loss" title="Loss of T140blocks">
+          <t>
+            In case of network failure or congestion, T140 data channels might get lost.
```

Comment:
Is 'get lost' the best term?  I have tried to find a good specification for it. looked in sdpneg, in data-channel and in W3C API.  "torn down with error indication" may be a suitable term. What do you think?

Reply:
I will look into it. I agree that "get lost" is not a good term.